python/grass/__init__.py

```diff
@@ -97,7 +147,7 @@ def _translate(text):
 # Initialize the translation attribute of the translate function to indicate
 # that the translations are not initialized.
 _translate.translation = None
-
+_import_user_settings()
 _builtins.__dict__["_"] = _translate
```

Comment:
If all you need is to call import in a function, _translate is already a function and it get called only when a translation is needed, not before that, so likely long after imports. Whatever is needed can be done there instead of the top-level. I understand you wrote the code before you saw #1838, but I think it is relevant.

Reply:
Yes, I understand how lazy translate work and I completely checked merged code from #1838 PR, before I update my PR. I can't import `UserSettings` class instance  inside `_translate()` function due to a circular import, check my first comment.